Source/Core/Core/HW/SI_GCAdapter.h

```diff
@@ -4,6 +4,8 @@
 
 #pragma once
 
+#include <libusb.h>
+
 #include "Common/Thread.h"
 #include "Core/HW/SI.h"
 #include "InputCommon/GCPadStatus.h"
```

Comment:
Adding this line here will get this patch compiling in Windows:

struct libusb_device;